app/src/main/main.ts

```diff
@@ -102,9 +102,9 @@ const createWindow = async () => {
 
   mainWindow = new BrowserWindow({
     show: false,
-    width: 1024,
+    width: 1200,
     height: 728,
-    minWidth: 1000,
+    minWidth: 1200,
```

Comment:
i may be unique, but i find this minWidth kinda annoying for my use cases on my laptop. I love shrinking windows to get to another windows (I don't do keyboard shortcuts.) I feel like its ok if it looks bad on small screens. i would rather be able to shrink to my liking.

Reply:
1150 is probably the smallest it can go without being omega gg